src/main/java/io/jenkins/plugins/designlibrary/Root.java

```diff
@@ -24,6 +31,22 @@ public String getUrlName() {
         return "design-library";
     }
 
+
+    @Override
+    public ContextMenu doContextMenu(StaplerRequest request, StaplerResponse response) throws Exception {
+        ContextMenu menu = new ContextMenu();
+        String url = Jenkins.get().getRootUrl() + getUrlName();
+        for (UISample s: getAll()) {
+            String iconFilename = s.getIconFileName() + " plugin-design-library";
+            menu.add(new MenuItem().withDisplayName(s.getDisplayName())
+                    .withIconClass(iconFilename)
+                    .withContextRelativeUrl(url + "/" + s.getUrlName())
```

Comment:
doesn't seem to work, when I run this locally in `hpi:run` I get this url when I click on the one of the samples:

http://localhost:8080/jenkins/http://localhost:8080/jenkins/design-library/File